release/mk-release-snapshots.sh

```diff
@@ -9,23 +9,42 @@
 SQUID_RELEASE=`ls -1 $SQUID_VCS_PATH | cut -d- -f2 | sort -h | tail -n 1`
 
 # Hostname of the CI server where snapshots are built
-SQUID_BUILD_SERVER=buildmaster.squid-cache.org
+SQUID_BUILD_SERVER=build.squid-cache.org
 
 # file Jenkins uses to assemble the list of snapshot files
 outfile=HEAD.out
 
+# arguments:
+# jenkins job name
+# destination directory
+get_artifacts() {
+	local job=$1
+	local outdir=$2
+	local artifactsUrl="https://$SQUID_BUILD_SERVER/job/${job}/lastSuccessfulBuild/artifact/artifacts/*zip*/artifacts.zip"
+	wget --quiet --ca-cert=/etc/ssl/certs/ISRG_Root_X1.pem "$artifactsUrl"
+	if [ ! -e artifacts.zip ] ; then
+		echo "could not download artifacts from $artifactsUrl"
+		return 1
+	fi
+	unzip -qq artifacts.zip
+	if [ ! -d artifacts ]; then
+		echo "${job}:artifacts.zip does not contain a subdirectory"
+		rm -d `unzip -Z1 artifacts.zip`
+		rm -f artifacts.zip
+		return 1
+	fi
+	mv artifacts/* $outdir
```

Comment:
Please handle mv errors.

Reply:
What exactly are you asking be done here?
AFAICS the items either move, or an error is logged and they stay where they are for later deletion.

Reply:
> What exactly are you asking be done here?

At the minimum, returning a non-zero result to indicate a failure. Personally, I would probably do more (e.g., the whole script should stop and exit with a failure instead of keep going as if nothing bad happened), but I am not asking for more.

Reply:
That means stopping just before cleanup. The next step is to do final cleanup whether this succeeds or fails. And relevant errors are already produced by the command itself.